domain-knowledge/BUSINESS-RULES.md

```diff
@@ -0,0 +1,548 @@
+
+Anybody may apply for an `Artist` profile. Doing so requires a basic `Account`,
+and once logged-in, the `User` must complete an `Artist` profile application.
+
+Service operator staff will review the application in a due-diligence capacity,
+looking for obvious signs of fraud and incorrect or incomplete information.
+
+The `User` will receive an email message upon approval and may begin creating
+`Albums` immediately thereafter.
+
+## Music Labels
+
+The premise behind music `Labels` is to make it simple for "record labels" to
+manage large music catalogs.
+
+### Applying for a Label Profile
+
+The process for applying for a `Label` profile is identical to that of applying
+for an `Artist` profile.
+
+### Adding a New Artist to a Label
+
+A `Label` may, at any time, add a new `Artist` to reside under its management.
+Doing so requires the `Label` to provide only the `Artist's` moniker
+("stage name"), and the `Label` can begin adding `Albums` under that `Artist`
+immediately.
+
+In this scenario, the `Label` exercises complete control over the `Artist`, which
+is in contrast to the next scenario, in which a `Label` invites an _existing_
+`Artist` who is controlled by a separate entity (person or business) to come
+under the `Label's` management.
+
+When a `Label` creates an `Artist`, a `CatalogEntity` is not created for the
+`Artist`, in which case the `Label's` `CatalogEntity` takes effect.
+
+#### Use Case
+
+When the `Artist` has no direct responsibility for managing its activities in
+relation to the service, i.e., the `Label` manages the `Artist` completely, it
+makes the most sense for the `Label` to create its `Artists` directly (rather
+than having `Artists` create their own `Accounts` and then inviting them to join
+the `Label`). This structure creates a lot less "paperwork" in that it eliminates
+redundant data entry and streamlines the application and approval process for
+each `Artist` on the `Label`.
+
+This configuration is best suited for larger `Labels` that manage sizable music
+catalogs and do not interact closely with their `Artists`.
+
+### Inviting an Existing Artist to Join a Label
+
+A `Label` may invite an existing `Artist` to join the `Label` at any time. Doing so only
+requires the `Label` to enter the`Artist's` email address. The `Artist` will
+receive an email message with further instructions.
+
+Upon accepting the invitation, the `Artist` thereby grants the `Label` certain
+managerial access to its `Account` and associated data.
+
+In this scenario, a `Label` is able to perform the following actions on behalf
+of an `Artist`:
+
+- Manage `Albums`, including any existing albums (Create, Read, Update, Delete)
+- Manage `Artist` `Profile` (Read, Update)
+- Override `CatalogEntity` (that is, the `Label` cannot view or change this
+  information; it can only override it, in which case its own `CatalogEntity`
+  takes precedence); among other things, this controls which `CatalogEntity` is
+  credited when a sale occurs
```

Comment:
So when the label's catalog entity takes precedence over the artist's they label is paid. How does the label pay the artist; is that out of the scope of the platform?

Reply:
Yes, exactly. The manner in which `Labels` pay their `Artists` is outside the scope of the platform.

The reasons for this are many (and complicated), including tax reporting requirements in any given jurisdiction, the `Artist's` preference with regard to how it's paid (electronic vs. paper check), specific contract and royalty arrangements, etc.

Reply:
Makes sense.